src/components/environments-summary/environment-card.js

```diff
@@ -64,7 +64,7 @@ const EnvironmentCard = ({ appName, env }) => {
     <div className="env_card">
       <div className="env_card_header">
         <div className="header">
-          <h6>
+          <Typography variant="accordion_header" group="ui">
```

Comment:
Is accordion_header a valid variant? Didn't see it in storybook, but perhaps storybook is outdated.

Reply:
accordion_header is a part of the UI group in EDS. See [here](https://github.com/equinor/design-system/tree/master/libraries/core-react/src/components/Typography) that we can use the group property to render any typography style. All typography styles can bee seen h[ere in figma](https://www.figma.com/file/0TbIXrrObWj80Cf7KucKYFL0/Design-Tokens?node-id=486%3A0&viewport=-3466%2C324%2C0.505901038646698).